.github/ISSUE_TEMPLATE/qa_issue_template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Manual QA
+about: Use this template for running manual QA issues.
+title: "[DATE]: [FEATURE NAME]"
+labels: qa, review
+---
+# Manual QA for # 
+
+### Check list
+- [] columns of experiments table are appropriate sizes 
```

Comment:
minor: let's try to use these new markup for the [task lists](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-tasklists)